src/main/java/piglatin/PigLatinConverter.java

```diff
@@ -40,7 +36,7 @@ public static String convertWordToPigLatin(String word) {
 	
 	
 	public static String convertStringToPigLatin(String inputString) {	
-		String[] words = inputString.split(" ");
+		String[] words = inputString.split("\\W+");
```

Comment:
Would have been nice to have this regex explained in a comment like "// Splits on all non-words of length >= 1"